msteams-platform/graph-api/activity-feed/feed-notifications.md

```diff
@@ -0,0 +1,171 @@
+> - Add a [webApplicationInfo](../../resources/schema/manifest-schema.md#webapplicationinfo) key to your app manifest with the following values:  
+&emsp;&emsp; &#8226; **id**  — your Azure AD app id.  
+&emsp;&emsp; &#8226;  **resource** — the resource url for the app.
+>
+> [!div class="checklist"]
+> [!div class="checklist"]
+>
+> - Add an **activities** key to your app manifest with an `activityType array`  value. Each activity type object will have the following properties:  
+&emsp;&emsp; &#8226; **type**  
+&emsp;&emsp; &#8226;  **description**  
+&emsp;&emsp; &#8226;  **templateText**
+
+```json
+"webApplicationInfo": {
+    "id": "73ee2834-38aa-4077-b4c9-e8e1c36f7e40",
+    "resource": "https://AkjStoreApp"
+  },
+  "activities": {
+    "activityTypes": [
+      {
+        "type": "taskCreated",
+        "description": "Task Created Activity",
+        "templateText": "{actor} created task {taskId} for you"
+      },
+      {
+        "type": "teamMention",
+        "description": "Team Mention Activity",
+        "templateText": "{actor} mentioned random team"
+      },
+      {
+        "type": "channelMention",
+        "description": "Channel Mention Activity",
+        "templateText": "{actor} mentioned random channel"
+      },
+      {
+        "type": "userMention",
+        "description": "Personal Mention Activity",
+        "templateText": "{actor} mentioned you dude"
+      },
+      {
+        "type": "calendarForward",
+        "description": "Forwarding a calendar event",
+        "templateText": "{actor} sent you an invite on behalf of {eventOwner}"
+      },
+      {
+        "type": "creatorTaskCreated",
+        "description": "Created Task Created",
+        "templateText": "The Creator created task {taskId} for you"
+      }
+    ]
+  },
+```
+
+## Activity feed notification payload properties
+
+| Property | Type | Required | Description |
+|-----|------|------|-----|
+|**topic**|teamworkActivityTopic | ✔ | represents what is being referenced in the feed item|
+|**activityType** | string | ✔| Represents the type of activity and must be declared in the [Teams App Manifest](../../resources/schema/manifest-schema.md)|
+|**recipient** | teamworkNotificationRecipient |  ✔ | The intended receiver. A recipient must be a Teams user with the ability to post notifications to everyone in a team, channel, and chat.|
+| **from** | string | |Displays a hint if the sender is different than the caller on the Graph token.|
+| **chainId** | long | | Enables the developer to override a previous notification. If not included, a new notifcation will be posted.|
+| **previewText** | itemBody | | Preview text displayed to the user as part an activity feed item. |
+| **teamsAppId** | long | | Creates a pointer to this specific Teams app.|
+| **templateParameters** | collection\<keyvaluepair\>| |Parameter values declared in the [Teams App Manifest](../../resources/schema/manifest-schema.md) |
+
+## Install your app directly in Teams
```

Comment:
Sideloading is only one way to install an app, we also support tenant (LOB) catalog today, and will support store submission. Same as RSC. :)